### ISSUE TITLE
User Task Activity

### ISSUE THREAD

**Issue by sfmskywalker** [OrchardCMS/OrchardCore#1707]
The User Task Activity allows a workflow user to define a number of actions that a user can perform on a content item when the workflow is blocked by this user task activity. These tasks are rendered as buttons on the content item editor. When the user clicks any of these buttons, the workflow resumes along the corresponding outcome associated with the button.

**Comment by sfmskywalker** [comment]
Duplicate of #1704
